### PR TITLE
Add User Metrics guide

### PR DESCRIPTION
**Pull request by userj**
An appdev guide related to User Metrics

### CODE REVIEW

**Comment by userj** [comment]
Wow, thanks for all the suggestions, and for taking a look. I'll slowly make the changes.

**Comment by userj** [comment]
I believe I have made the requested changes. Do I need to do another PR?

**Comment by UniversalSuperBox** [comment]
No need, the changes are made here as well.

File: appdev/guides/user-metrics.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+(There may be updated versions of this above 0.1)
+
+Next, the specific Metric must be defined in the code as an object:
+
+.. code:: qml
+
+            Metric { // Define the Metric object.
+                property string circleMetric // Create a string-type variable called “circleMetric”. This is so you can update it later from somewhere else.
+                id: metric // Give the metric a name/id. Here it’s called “metric”
```
Comment (UniversalSuperBox) [inline appdev/guides/user-metrics.rst]: This explanation of "id" seems like too much information for a seasoned QML developer, not enough for a new one... In short, a qml object's ID is how you reference it elsewhere in the code.

**Pull request merged**